### PR TITLE
Added step definition to "wait for element to be gone"

### PR DESCRIPTION
**Pull request by pookmish**
# READY FOR REVIEW

# Summary
- Added step definition to `Then I wait for element ".this-is-a-class" to be gone`

# Need Review By (Date)
- asap

# Urgency
- medium

# Steps to Test
1. try out the step if you'd like

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)

### CODE REVIEW

File: src/StanfordBehat/DrupalExtension/Context/SwsMinkContext.php [hunk 1]
```diff
@@ -139,15 +140,31 @@ public function iWaitForElement($selector) {
     }, $selector);
   }
 
+  /**
+   * Wait up to 5 seconds for an element to be visible.
```
Comment (jbickar) [inline src/StanfordBehat/DrupalExtension/Context/SwsMinkContext.php]:
```suggestion
   * Wait up to 5 seconds for an element to be no longer present.
```

File: src/StanfordBehat/DrupalExtension/Context/SwsMinkContext.php [hunk 2]
```diff
@@ -162,7 +179,7 @@ protected function waitForSelector(callable $lambda, $selector, $attempts = 30)
       sleep(1);
     }
 
-    throw new ElementNotVisible('The element is not available ' . $last_message);
+    throw new ElementNotVisible("The element is$not available $last_message");
```
Comment (jbickar) [inline src/StanfordBehat/DrupalExtension/Context/SwsMinkContext.php]:
```suggestion
    throw new ElementNotVisible("The element is $not available $last_message");
```

**Review by jbickar — CHANGES_REQUESTED** [review]
Looks good per https://github.com/SU-SWS/acsf-cardinalsites/pull/123; small tweaks for readability.

File: src/StanfordBehat/DrupalExtension/Context/SwsMinkContext.php (hunk 2 above)
Comment (pookmish) [reply src/StanfordBehat/DrupalExtension/Context/SwsMinkContext.php]: having the space produces double spaces sometimes if `$not = ''`

Comment (pookmish) [reply src/StanfordBehat/DrupalExtension/Context/SwsMinkContext.php]: not really a big deal though. :man_shrugging:

**Review by jbickar — APPROVED** [review]
GTG

File: src/StanfordBehat/DrupalExtension/Context/SwsMinkContext.php (hunk 2 above)
Comment (jbickar) [reply src/StanfordBehat/DrupalExtension/Context/SwsMinkContext.php]: Ah right. Ah well. Whatevs.

**Pull request merged**